### PR TITLE
feat: possibility to create a koson array from a collection

### PR DESCRIPTION
**Pull request by achatellier**
... and fix tests on windows

### CODE REVIEW

**Comment by achatellier** [comment]
Regarding the line separator, I understand the need to select the right separator when koson produce some data. 
But when processing data from the outside world, koson cannot make assumption about the separator used in strings.

File: src/main/kotlin/com/lectra/koson/Koson.kt [hunk 1]
```diff
@@ -3,6 +3,9 @@ package com.lectra.koson
 @Suppress("ClassName")
 object arr : ArrayType() {
     operator fun get(vararg elements: Any?): ArrayType =
+            invoke(elements.asList())
+
+    operator fun invoke(elements: Collection<Any?>): ArrayType =
```
Comment (ncomet) [inline src/main/kotlin/com/lectra/koson/Koson.kt]: arr() would kind of break the DSL syntax, I suggest
```
   operator fun get(elements: Iterable<Any?>): ArrayType =
        ArrayType(elements.map { toAllowedType(it) }.toList())
```
providing the ability to do

`arr[...]` or `arr[listOf(...)]`

Comment (ncomet) [reply src/main/kotlin/com/lectra/koson/Koson.kt]: And `Iterable` because it is more open

File: src/test/kotlin/com/lectra/koson/KosonTest.kt [hunk 2]
```diff
@@ -540,7 +542,7 @@ class KosonTest {
         }
 
         @Test
-        fun `array with rawjson must be inlined properly`() {
+        fun `array with rawjson must be inlined properly in unix format`() {
```
Comment (ncomet) [inline src/test/kotlin/com/lectra/koson/KosonTest.kt:568]: We also need to test object and array for windows carriage returns inputs

**Review by ncomet — CHANGES_REQUESTED** [review]
I put some annotations on the files with suggestions, then we can merge ! Thanks !

**Review by ncomet — APPROVED** [review]

**Pull request merged**